exposeConfig with specific part is undefined

### Version
@nuxtjs/tailwindcss: **5.0.3**
nuxt: **2.15.8**

### Steps to reproduce

Set `exposeConfig` to `true` in `nuxt.config.js`
```js
export default {
  tailwindcss: {
    exposeConfig: true
  }
}
```

Import `theme` from configuration in component.
```js
import { theme } from '#tailwind-config'
```


### What is Expected?
Expecting `theme` to be defined.


### What is actually happening?
`theme` is undefined.

When importing the full configuration, I can get the `theme` field.

Hi, 

**VERSION**
```json
{
  "@nuxtjs/tailwindcss": "^5.1.3",
  "typescript": "^4.7.4"
  "nuxt": "3.0.0-rc.4",
}
```

**STEPS TO REPRODUCE**
Same steps as HZooly did :

Set `exposeConfig` to `true` in `nuxt.config.ts`.

When trying to create util method, I'm importing theme the way the doc says :

```typescript
import { theme } from '#tailwind-config'
```

I also tried with : 
```typescript
import { theme } from '~/tailwind-config'
import { theme } from '../tailwind.config.js'
```

or even :
```typescript
import resolveConfig from 'tailwindcss/resolveConfig';
import tailwindConfig from '../../../tailwind.config.js';

const fullConfig = resolveConfig(tailwindConfig)
```

Finally, I tried with installing `@types/tailwindcss` and doing this way :

```typescript
import { TailwindConfig } from 'tailwindcss/tailwind-config';
import resolveConfig from 'tailwindcss/resolveConfig'
import tailwindConfig from '../tailwind.config.js'

const fullConfig = resolveConfig(tailwindConfig as unknown as TailwindConfig)
```

Then got this error on runtime :

```shell
TS2307: Cannot find module '#tailwind-config' or its corresponding type declarations.
```

or on devtools console : 
```shell
Uncaught (in promise) SyntaxError: The requested module '/_nuxt/tailwind.config.js' does not provide an export named 'default' (at colors.ts:2:1)
```


I don't know if I'm doing this the wrong way but it seems I'm not alone with this issue.

--- a/src/module.ts
+++ b/src/module.ts
@@ -150,7 +150,7 @@ export default defineNuxtModule<ModuleOptions>({
     if (moduleOptions.exposeConfig) {
       const template = addTemplate({
         filename: 'tailwind.config.mjs',
-        getContents: () => `${Object.entries(resolvedConfig).map(([k, v]) => `export const ${k} = ${JSON.stringify(v, null, 2)}`).join('\n')}\nexport default ${JSON.stringify(resolvedConfig, null, 2)}`
+        getContents: () => `${Object.entries(resolvedConfig).map(([k, v]) => `export const ${k} = ${JSON.stringify(v, null, 2)}`).join('\n')}\nexport default {${Object.keys(resolvedConfig)}}`
       })
       addTemplate({
         filename: 'tailwind.config.d.ts',

--- a/src/module.ts
+++ b/src/module.ts
@@ -150,7 +150,7 @@ export default defineNuxtModule<ModuleOptions>({
     if (moduleOptions.exposeConfig) {
       const template = addTemplate({
         filename: 'tailwind.config.mjs',
-        getContents: () => `${Object.entries(resolvedConfig).map(([k, v]) => `export const ${k} = ${JSON.stringify(v, null, 2)}`).join('\n')}\nexport default {${Object.keys(resolvedConfig)}}`
+        getContents: () => `${Object.entries(resolvedConfig).map(([k, v]) => `export const ${k} = ${JSON.stringify(v, null, 2)}`).join('\n')}\nexport default { ${Object.keys(resolvedConfig).join(', ')} }`
       })
       addTemplate({
         filename: 'tailwind.config.d.ts',

--- a/src/module.ts
+++ b/src/module.ts
@@ -150,7 +150,7 @@ export default defineNuxtModule<ModuleOptions>({
     if (moduleOptions.exposeConfig) {
       const template = addTemplate({
         filename: 'tailwind.config.mjs',
-        getContents: () => Object.entries(resolvedConfig).map(([k, v]) => `export const ${k} = ${JSON.stringify(v)}`).join('\n')
+        getContents: () => `${Object.entries(resolvedConfig).map(([k, v]) => `export const ${k} = ${JSON.stringify(v, null, 2)}`).join('\n')}\nexport default ${JSON.stringify(resolvedConfig, null, 2)}`
       })
       addTemplate({
         filename: 'tailwind.config.d.ts',

--- a/src/module.ts
+++ b/src/module.ts
@@ -150,7 +150,8 @@ export default defineNuxtModule<ModuleOptions>({
     if (moduleOptions.exposeConfig) {
       const template = addTemplate({
         filename: 'tailwind.config.mjs',
-        getContents: () => `export default ${JSON.stringify(resolvedConfig, null, 2)}`
+        getContents: () => `export default ${JSON.stringify(resolvedConfig, null, 2)}`,
+        write: true
       })
       addTemplate({
         filename: 'tailwind.config.d.ts',

--- a/src/module.ts
+++ b/src/module.ts
@@ -148,13 +148,14 @@ export default defineNuxtModule<ModuleOptions>({
     // Expose resolved tailwind config as an alias
     // https://tailwindcss.com/docs/configuration/#referencing-in-javascript
     if (moduleOptions.exposeConfig) {
+      const configOptions = Object.keys(resolvedConfig)
       const template = addTemplate({
         filename: 'tailwind.config.mjs',
-        getContents: () => `${Object.entries(resolvedConfig).map(([k, v]) => `export const ${k} = ${JSON.stringify(v, null, 2)}`).join('\n')}\nexport default { ${Object.keys(resolvedConfig).join(', ')} }`
+        getContents: () => `${Object.entries(resolvedConfig).map(([k, v]) => `export const ${k} = ${JSON.stringify(v, null, 2)}`).join('\n')}\nexport default { ${configOptions.join(', ')} }`
       })
       addTemplate({
         filename: 'tailwind.config.d.ts',
-        getContents: () => 'declare const config: import("tailwindcss").Config\nexport { config as default }',
+        getContents: () => `type tailwindcssConfig = import("tailwindcss").Config\ndeclare const config: tailwindcssConfig\n${configOptions.map((o) => `declare const ${o}: tailwindcssConfig["${o}"]`).join('\n')}\nexport { config as default, ${configOptions.join(', ')} }`,
         write: true
       })
       nuxt.options.alias['#tailwind-config'] = template.dst

--- a/src/module.ts
+++ b/src/module.ts
@@ -150,8 +150,7 @@ export default defineNuxtModule<ModuleOptions>({
     if (moduleOptions.exposeConfig) {
       const template = addTemplate({
         filename: 'tailwind.config.mjs',
-        getContents: () => `export default ${JSON.stringify(resolvedConfig, null, 2)}`,
-        write: true
+        getContents: () => Object.entries(resolvedConfig).map(([k, v]) => `export const ${k} = ${JSON.stringify(v)}`).join('\n')
       })
       addTemplate({
         filename: 'tailwind.config.d.ts',

